### PR TITLE
SEO-176402-NET-MAUI-Text-Input-Layout-Int

### PR DESCRIPTION
**Pull request by berylouma**
|Title|Description|
|--|--|
|Task Category| .NET MAUI TextInputLayout|
|Content Task Link/Mail Screenshot| NA|
|UX task| NA|
|Interlinking PR link| Int -https://github.com/syncfusion-content/maui-docs/pull/1879|
|Ticket/Task link| https://dev.azure.com/Syncfusion-Infrastructure/Syncfusion%20SEO/_workitems/edit/179663/|
|Excel/SharePoint link| https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/NET-MAUI-Text-Input-Layout--Money-Keywords.aspx|

Worked on 17 keywords in the .NET MAUI Text Input Layout and am keeping track of all keyword details in the link below:-
[https://syncfusion.sharepoint.com/sites/GH/SEO/SitePages/Keyword-Plan---.NET-MAUI-Text-Input-Layout.aspx]

### CODE REVIEW

File: MAUI/ListView/selection.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ listView.SelectedItems.Add (viewModel.MusicInfo[5]);
 {% endhighlight %}
 {% endtabs %}
 
-N> The `SfListView.SelectedItems` property type is a type of ObservableCollection<Object>. So, you should use generic type as an object for the ViewModel collection property type.
+N> The `SfListView.SelectedItems` property is of type `ObservableCollection<Object>`. Therefore, it is recommended to utilize a ViewModel collection property with the same type, `ObservableCollection<Object>`, to ensure proper functioning of the `SelectedItems` property.
```
Comment (Sharly-Atieno) [inline MAUI/ListView/selection.md:104]: Revert this

File: MAUI/TextInputLayout/Getting-Started.md [hunk 2]
```diff
@@ -161,4 +161,4 @@ N> Password visibility toggle can be enabled only for [Entry](https://learn.micr
 
 You can find the complete getting started sample from this [link.](https://github.com/SyncfusionExamples/maui-textinputlayout-samples)
 
-
+N> You can refer to our [.NET MAUI TextInputLayout](https://www.syncfusion.com/maui-controls/maui-textinputlayout) feature tour page for its groundbreaking feature representations. You can also explore our [.NET MAUI TextInputLayout Example](https://github.com/syncfusion/maui-demos/tree/master/MAUI/TextInputLayout) that shows you how to render the TextInputLayout in .NET MAUI.
```
Comment (Sharly-Atieno) [inline MAUI/TextInputLayout/Getting-Started.md:164]: Use as below in all places
[.NET MAUI Text Input Layout]

File: MAUI/ImageEditor/zooming.md [hunk 3]
```diff
@@ -41,6 +41,8 @@ To enable or disable the zooming functionality, set the value of the [`AllowZoom
 
 Programmatically, you can zoom the loaded image without any interaction by setting a value to the [`ZoomLevel`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_ZoomLevel) property. 
 
+N> The [`ZoomLevel`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfImageEditor.html#Syncfusion_Maui_ImageEditor_SfImageEditor_ZoomLevel) property of the image editor will be updated based on the image zoomed status.
```
[the remark this answers is not in the document]
Comment (berylouma) [reply MAUI/ImageEditor/zooming.md]: Reverted

File: MAUI/TextInputLayout/Getting-Started.md (hunk 2 above)
Comment (berylouma) [reply MAUI/TextInputLayout/Getting-Started.md]: Done

File: MAUI/ListView/selection.md (hunk 1 above)
Comment (berylouma) [reply MAUI/ListView/selection.md]: Reverted

**Review by Sharly-Atieno — APPROVED** [review]
Approved

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**